### DISCUSSION TITLE
Missing database tables

### DISCUSSION THREAD

**Discussion by majchinthaka** [spatie/laravel-backup#1392] [Q&A]
When I run the 'php artisan backup:run' command or 'php artisan backup:run --only-db' command, the .sql backup file which is in the some database tables have been missing.

**Comment by erikn69** [comment]
Version of package?
Look #1389 https://github.com/spatie/laravel-backup/discussions/1377#discussioncomment-1207602

**Reply by majchinthaka** [reply]
version 6.16

**Reply by erikn69** [reply]
Try with PR #1386 or upgrade to 7.x
